lib/src/publisher.dart

```diff
@@ -61,22 +58,64 @@ class Resource {
   Resource(this.generator);
 }
 
+/// Storage of all published resources. The general idea behind publishing
+/// resources is as follows. One may publish a resource under some name on
+/// server side, associating it with some [DataGenerator], which is a way of
+/// creating [DataProvider] from some parameters. One may also subscribe to this
+/// resource on client side (using [Subscriber] and [Subscription]). This starts
+/// an automatic machinery that keeps the client-side collection of data in sync
+/// with the collection provided by the published resource.
+///
+/// Example:
```

Comment:
This example would be easier to understand if there actually was an example of code.